sdk/appconfiguration/azure-data-appconfiguration-v2/src/main/java/com/azure/v2/data/appconfiguration/models/FeatureFlagConfigurationSetting.java

```diff
@@ -118,8 +118,8 @@ public String getValue() {
             newValue = outputStream.toString(StandardCharsets.UTF_8.name());
             outputStream.close();
         } catch (IOException exception) {
-            LOGGER.logThrowableAsError(
-                new IllegalArgumentException("Can't parse Feature Flag configuration setting value.", exception));
+            throw LOGGER.throwableAtError()
```

Comment:
was it intentional that we didn't threw here? looks like a bug

Reply:
Can't speak for Shawn, but seemed like we dont throw with [GAed App Config ](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/FeatureFlagConfigurationSetting.java#L122) either/

Reply:
discussed offline, seems like a bug